s3am/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:12.04
+
+MAINTAINER Frank Austin Nothaft
+
+# install necessary dependencies
+RUN apt-get update && \
+    apt-get install -y python-dev \
+            gcc \
+            make \
+            libcurl4-openssl-dev \
+            python-virtualenv
+
+# create virtualenv and install s3am
+RUN virtualenv /opt/s3am && . /opt/s3am/bin/activate
+RUN pip install s3am
```

Comment:
I propose using the latest build (it's still a beta but has important fixes in it) and mentioning the concrete version, here and in the `tag=` line in the Makefile.

`pip install --pre s3am==1.0b1.dev28`